src/_guides/libraries/useful-libraries.md

```diff
@@ -21,146 +21,121 @@ This document talks about the first two kinds of libraries,
+| [mockito](https://pub.dartlang.org/packages/mockito) | Mockito is a popular framework for developing unit tests.  The Dart version of the Mockito library is especially useful if you are writing tests for dependency injection. | Answering, Expectation, Verification |
+| [path](https://pub.dartlang.org/packages/path) | Provides common operations for manipulating paths on different OS platforms. See [Unboxing Packages: path](http://news.dartlang.org/2016/06/unboxing-packages-path.html) for more information. | absolute, basename, extension, normalize, split |
+| [quiver](https://pub.dartlang.org/packages/quiver) | A set of utility libraries that makes it easier and more convenient to use many Dart libraries. Some of the libraries where Quiver provides additional support include async, cache, collection, core, iterables, patterns, and testing. | CountdownTimer, FutureGroup, FutureStream (quiver.async), MapCache (quiver.cache), MultiMap, TreeSet (quiver.collection), EnumerateIterable (quiver.iterables), center, compareIgnoreCase, isWhiteSpace (quiver.strings)  |
+| [shelf](https://pub.dartlang.org/packages/shelf) | Web server middleware for Dart. Shelf makes it easy to create and compose web servers, and parts of web servers. | Cascade, Pipeline, Request, Response, Server |
+| [stagehand](https://pub.dartlang.org/packages/stagehand) | A Dart project generator. WebStorm and IntelliJ use these templates when you create a new application, but you can also use the templates from the command line via [pub global activate](https://www.dartlang.org/tools/pub/cmd/pub-global). | Templates include console-full, console-simple, server-shelf, web-angular, and web-simple. |
+| [test](https://pub.dartlang.org/packages/test) | A standard way of writing and running tests in Dart. This package replaces the deprecated unittest package. | expect, group, test |
+{:.table .table-striped .nowrap}
 
 Search for other available packages on [pub.dartlang.org](https://pub.dartlang.org/).
 
-## Importing libraries and packages
+## Specialized libraries
+
+This page doesn't include some of the more specialized libraries that are covered
+elsewhere. Learn more about the following topics using the provided links:
+
+### Web libraries
+
+If you write web apps, you should know about Angular 2, a web application framework.
+You might try a [code lab for Angular Dart](https://webdev.dartlang.org/codelabs/ng2),
+or refer to the [Angular Dart docs](https://angular.io/docs/dart/latest/).
+Other available resources are the
+[js](https://www.dartdocs.org/documentation/js/latest) package
+for interoperability with JavaScript APIs, and the
+[dart:html](https://api.dartlang.org/stable/dart-html/dart-html-library.html)
+library for low-level HTML programming.
+For more information about Dart on the Web, see [webdev.dartlang]({{site.webdev}}).
```

Comment:
Should the link anchor text be webdev.dartlang.org instead of 'webdev.dartlang'?